### PR TITLE
remove servicecatalog test from master

### PR DESCRIPTION
**Pull request by jmrodri**
Fix failed explain job from openshift-svcat-*-operator tests.

### CODE REVIEW

**Comment by jmrodri** [comment]
Blocks 4.5 feature https://prow.svc.ci.openshift.org/view/gcs/origin-ci-test/pr-logs/pull/openshift_cluster-svcat-apiserver-operator/74/pull-ci-openshift-cluster-svcat-apiserver-operator-master-e2e-aws/229

File: test/extended/cli/explain.go [hunk 1]
```diff
@@ -119,8 +119,8 @@ var (
 		{Group: "operator.openshift.io", Version: "v1", Resource: "kubeschedulers"},
 		{Group: "operator.openshift.io", Version: "v1", Resource: "openshiftcontrollermanagers"},
 		{Group: "operator.openshift.io", Version: "v1", Resource: "servicecas"},
-		{Group: "operator.openshift.io", Version: "v1", Resource: "servicecatalogapiservers"},
-		{Group: "operator.openshift.io", Version: "v1", Resource: "servicecatalogcontrollermanagers"},
+		// {Group: "operator.openshift.io", Version: "v1", Resource: "servicecatalogapiservers"},
```
Comment (jwforres) [inline test/extended/cli/explain.go]: is there a reason to comment them vs just removing them?

Comment (jmrodri) [reply test/extended/cli/explain.go]: @jwforres @shawn-hurley no, I saw others were commented out :) I'll remove them instead.

**Pull request merged**